react/features/base/tracks/reducer.js

```diff
@@ -133,3 +134,17 @@ ReducerRegistry.register('features/base/tracks', (state = [], action) => {
         return state;
     }
 });
+
+/**
+ * Listen for actions that mutate the no-src-data state, like the current notification id
+ */
+ReducerRegistry.register('features/base/no-src-data', (state = {}, action) => {
```

Comment:
This doesn't feel exactly kosher ( having to reducer states in the same file), however the alternative would be to move the no src data notification functionality to a feature of its own. I didn't know what the best approach here was so waiting for suggestions. Thanks.